# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/warp/bug1762769.js
# Commit: 0ba054ab3339
# Full Hash: 0ba054ab33393a437711d2746a2111206b08e0db
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2022-04-12 03:57:01
# Description:
#   Bug 1762769: Don't mark ops only reachable via retsub as normally reachable r=jandem
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D143095
# ==============================================================================

diff -r 5ba291945296 -r 0ba054ab3339 js/src/jit-test/tests/warp/bug1762769.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/warp/bug1762769.js	Mon Apr 11 19:28:30 2022 +0000
@@ -0,0 +1,8 @@
+try {
+  try {}
+  finally {
+    throw 1;
+  }
+} catch {}
+
+for (var i = 0; i < 1000; i++) {}